src/main/java/core/basesyntax/AllColors.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+public enum AllColors {
```

Comment:
How do you think, which name better for this enum: "AllColors" or simpler "Colors"?